website/docs/why-ledokku.md

```diff
@@ -0,0 +1,27 @@
+---
+id: why-ledokku
+title: Why Ledokku?
+---
+
+### Ledokku in comparison to other tools
+
+_*Name sounds cool, visuals are great, but how Ledokku is different from other deployment platforms?*_ This so far
+has been the most common reaction from people who have been introduced to Ledokku.
+
+Before we dive into comparisons, couple of disclaimers: _We at Ledokku ❤️ Vercel and Netlify as for our platform we use Next.js which is framework created by Vercel and our docs are hosted on Netlify._
+
+### Vercel
+
+Ledokku is more flexible than Vercel as it allows you to deploy apps written in any of the most popular programming languages and link them any of the most popular databases. Where as Vercel is focusing just on JAMstack static apps and certainly does it really well.
```

Comment:
vercel is focusing on JAMstack and serverless if I am correct :)

Reply:
@pradel added.